zeebe/gateway-grpc/src/test/java/io/camunda/zeebe/gateway/interceptors/InterceptorIT.java

```diff
@@ -118,7 +118,7 @@ void shouldAbortDeploymentCalls() {
 
     // when
     gateway.start().join();
-    try (final var client = createZeebeClient()) {
```

Comment:
🔧 ❌  The CI is failing due to a wrong import for `DeploymentEvent`

Reply:
🙈